pkg/render/logstorage/linseed/linseed.go

```diff
@@ -254,6 +279,7 @@ func (l *linseed) linseedDeployment() *appsv1.Deployment {
 			ValueFrom: secret.GetEnvVarSource(render.ElasticsearchLinseedUserSecret, "password", false),
 		},
 		{Name: "ELASTIC_CA", Value: l.cfg.TrustedBundle.MountPath()},
+		{Name: "MULTI_TENANT", Value: fmt.Sprintf("%v", l.cfg.MultiTenant)},
```

Comment:
Linseed should just check the EXPECTED_TENANT_ID option to determine if it's running in a multi-tenant cluster.